### PR TITLE
✨ new scrna qc docker

### PR DESCRIPTION
**Pull request by migbro**
## Description
Based on https://github.com/hbctraining/scRNA-seq_online/blob/master/README.md#installation-requirements, new image for scRNA QC

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Last step of workflow here: https://cavatica.sbgenomics.com/u/d3b-bixu/kf-dev-single-cell-rna/tasks/f7ed86d4-2696-401f-a025-c915d39a6ee2/
- [ ] Also tested locally as an interactive image running https://github.com/kids-first/kf-dev-single-cell-rna/blob/feature/solo-multipmap/scripts/seurat_hbc_scrna_qc.R
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ gatk|4.beta.6|docker pull pgc-images.sbgenomics.com/d3b-bixu/gatk:4.beta.6
 gatk|4.beta.6-tabix|docker pull pgc-images.sbgenomics.com/d3b-bixu/gatk:4.beta.6-tabix
 gatk|latest|docker pull pgc-images.sbgenomics.com/d3b-bixu/gatk:latest
 gistic|latest|docker pull pgc-images.sbgenomics.com/d3b-bixu/gistic:latest
+hbc_scrna_qc|v1.0.0-scrnaseq|docker pull pgc-images.sbgenomics.com/d3b-bixu/hbc_scrna_qc:v1.0.0-scrnaseq
```
Comment (dmiller15) [inline README.md:85]: I think you should just go with version `scrnaseq` even though that's dumb.

File: hbc_scrna_qc/v1.0.0-scrnaseq/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM rocker/tidyverse:4.4.0
+LABEL maintainer="Miguel Brown (brownm28@chop.edu)"
+LABEL description="Docker image to run Harvard Bioinformatics Core scRNA QC. Install specific version of Seurat for downstream compatibility"
```
Comment (dmiller15) [inline hbc_scrna_qc/v1.0.0-scrnaseq/Dockerfile:3]:
```suggestion
LABEL description="Docker image to run Harvard Bioinformatics Core (HBC) scRNA QC as defined here: https://github.com/hbctraining/scRNA-seq_online?tab=readme-ov-file#installation-requirements. Install specific version of Seurat for downstream compatibility"
```

**Review by dmiller15 — CHANGES_REQUESTED** [review]
Need to establish a concrete connection to the software you are referencing in the image name.

File: README.md (hunk 1 above)
Comment (migbro) [reply README.md:85]: I thought deviating from semantic versioning would break the GHA. Maybe I over thought that.

**Review by dmiller15 — APPROVED** [review]

**Review by wongjessica93 — APPROVED** [review]

**Pull request merged**